### PR TITLE
ECCW-529: Hotjar cookie + js removal, disabling.

### PR DESCRIPTION
**Pull request by siliconmeadow**

### CODE REVIEW

File: config/default/eu_cookie_compliance.settings.yml [hunk 1]
```diff
@@ -54,9 +54,9 @@ cookie_name: ''
 exclude_uid_1: false
 better_support_for_screen_readers: false
 method: categories
-disabled_javascripts: 'google_analytics:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js'
+disabled_javascripts: "analytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js\r\nanalytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-hotjar-script-runner.js"
 automatic_cookies_removal: true
-allowed_cookies: "analytics_cookies:_ga*\r\nanalytics_cookies:_ga"
+allowed_cookies: "analytics_cookies:_ga*\r\nanalytics_cookies:_ga\r\nanalytics_cookies:_hj*\r\nanalytics_cookies:NEXT_LOCALE\r\nanalytics_cookies:ps_mode\r\nanalytics_cookies:Optanon*\r\nanalytics_cookies:_BEAMER*\r\nanalytics_cookies:_gcl_au\r\nanalytics_cookies:acq_homepage*"
```
Comment (MatthewWilkes) [inline config/default/eu_cookie_compliance.settings.yml:59]: Hi @siliconmeadow 

Some of these are surprising to me. The page at https://help.hotjar.com/hc/en-us/articles/6952777582999-Cookies-Set-by-the-Hotjar-Tracking-Code shows only cookies starting `_hj*` and `hj*`. What are `NEXT_LOCALE`, `ps_mode`, `Optanon*`, `_BEAMER`, `_gcl_au` and `acq_homepage*`?

Comment (siliconmeadow) [reply config/default/eu_cookie_compliance.settings.yml:59]: While testing hotjar, I set up actions and found those extra cookies being set:

image

They're mentioned here as functional cookies:

https://help.hotjar.com/hc/en-us/articles/115011789248

Comment (siliconmeadow) [reply config/default/eu_cookie_compliance.settings.yml:59]: Sorry. I see the err of my ways re: 3rd party cookies.

File: config/default/eu_cookie_compliance.settings.yml [hunk 2]
```diff
@@ -54,9 +54,9 @@ cookie_name: ''
 exclude_uid_1: false
 better_support_for_screen_readers: false
 method: categories
-disabled_javascripts: 'google_analytics:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js'
+disabled_javascripts: "analytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js\r\nanalytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-hotjar-script-runner.js"
```
Comment (MatthewWilkes) [inline config/default/eu_cookie_compliance.settings.yml:57]: @siliconmeadow We're not using the hotjar script, we're bringing it in with GTM. Same with the ga runner, actually.

**Review by MatthewWilkes — APPROVED** [review]
Great, thanks

**Pull request merged**